src/Admin/AbstractAdminExtension.php

```diff
@@ -127,11 +127,6 @@ public function configureDashboardActions(AdminInterface $admin, array $actions)
         return $actions;
     }
 
-    public function configureDashboardButtons(AdminInterface $admin, array $actions): array
-    {
-        return $actions;
-    }
-
```

Comment:
Can we remove it without upgrade note? Or is it done in other place?

Reply:
This was deprecated from the interface in 3.x